### PR TITLE
LPS-192140 Add cases for multiselect and picklist

### PR DESCRIPTION
**Pull request by victorg1991**
Only the last one needs review, the other commits are already waiting to be merged here:https://github.com/brianchandotcom/liferay-portal/pull/138654

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/info/info-test/build.gradle [hunk 1]
```diff
@@ -1,4 +1,6 @@
 dependencies {
+	compileOnly project(":apps:list-type:list-type-api")
```
Comment (ealonso) [inline modules/apps/info/info-test/build.gradle:2]: @victorg1991 It should be testIntegrationCompile instead of compileOnly

**Pull request closed without merging**